applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1165,14 +1165,23 @@ int le_audio_play_pause(void)
 	return 0;
 }
 
-int le_audio_send(uint8_t const *const data, size_t size)
+int le_audio_send(struct encoded_audio enc_audio)
 {
 	int ret;
 	struct bt_iso_tx_info tx_info = { 0 };
-	size_t sdu_size = LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_BT_AUDIO_BITRATE);
+	size_t data_size_pr_stream;
 
-	if (size != (sdu_size * 2)) {
-		LOG_ERR("Not enough data for stereo stream");
+	if (enc_audio.num_ch == 1) {
+		data_size_pr_stream = enc_audio.size;
+	} else if (enc_audio.num_ch == 2) {
```

Comment:
Hmm, here we assume that CIS gateway has either 1 or 2 streams (not actually checking number of streams)
Should this be made more flexible?

Reply:
Changed